src/commands/deployImage/imageSource/containerRegistry/acr/createAcr/ICreateAcrContext.ts

```diff
@@ -0,0 +1,14 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { KnownSkuName, Registry } from "@azure/arm-containerregistry";
+import { IResourceGroupWizardContext } from "@microsoft/vscode-azext-azureutils";
+import { ExecuteActivityContext } from "@microsoft/vscode-azext-utils";
+
+export interface ICreateAcrContext extends IResourceGroupWizardContext, ExecuteActivityContext {
+    newRegistryName?: string;
+    sku?: KnownSkuName;
```

Comment:
Nit: Add the new prefix to sku - `newRegistrySku`

Reply:
Is there a specific reason we are being super verbose and adding new to the property names here?

Reply:
I just noticed we do that in a ton of places to differentiate between an existing resource vs. information gathering for creating a new one.  I figured it was kind of an unspoken convention, but happy to not do it that way as well

Reply:
I was just curious if there was a place where not having the new specified was confusing. But I think it's fine. It can't hurt to add more context to the property name.